crates/astria-sequencer/TESTNET.md

```diff
@@ -28,37 +28,51 @@ This will create 3 cometbft validator configs and put the validator configs in
 `./sequencer_testnet` by default. It will then start the respective sequencer
 applications (one for each cometbft validator) and the cometbft validator nodes.
 
+(This requires cometbft to be installed)
+
 ```sh
-./scripts/testnet.sh
+just run-testnet
```

Comment:
i liked the instructions better before, this is gonna rebuild every time (even though it should be fast but still)

Reply:
Cargo doesn't rebuild when things are cached and there are no changes, no?